src/commands/createTransactionRegisterMultisignatureAccount.js

```diff
@@ -28,21 +28,21 @@ const description = `Creates a transaction which will register a multisignature
 	Minimum as the amount of signatures needed until the transaction will be processed.
 
 	Examples:
-	- create transaction create multisignature account 24 2 215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca 922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa
+	- create transaction register multisignature account 24 2 215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca 922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa
 	- create transaction 4 24 2 215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca 922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa
 `;
 
 const processInputs = (lifetime, minimum, keysgroup) => ({
 	passphrase,
 	secondPassphrase,
 }) =>
-	transactions.createMultisignature(
-		passphrase,
-		secondPassphrase,
+	transactions.registerMultisignature({
+		secret: passphrase,
```

Comment:
In lisk-js 1.0.0 it has now changed to be the `passphrase` aswell instead of `secret`. Let's adapt this and upgrade the 1.0.0-dev-lisky branch

Reply:
Also in the other transaction generators